### ISSUE TITLE
Crash when copy paste from LibreOffice Writer

### ISSUE THREAD

**Issue by pavreh** [vslavik/poedit#495]
Poedit crashes when copy text from Writer and paste it to the translation filed in Poedit.

1) Open Writer
2) Type word "hello"
3) Select the word then Ctrl+C
4) Open a .po file in Poedit
5) Click to the translation field then Ctrl+V
Blank dialog window appears. Click on the close button do nothing. Poedit itself does not response.

Poedit version 2.0.8
Writer version 6.0.3.2
Ubuntu minimal 18.04 + GNOME

Copy paste from a text editor (Gedit) is functional.

**Comment by vslavik** [comment]
Ubuntu 18.04 contains Poedit 2.0.6, not 2.0.8. Seems like you compiled from sources and forgot to mention that crucial fact (or all the other accompanying information, like wxGTK version, which GTK+ version it uses etc)?

**Comment by vslavik** [comment]
Sorry, let me rephrase: Can you please try with the official Ubuntu package (as it seems you self-compiled, because 2.0.6 is the latest Ubuntu-packaged)? The intention is ruling out possible miscompilation issues or issues with your version of wxGTK etc.

**Comment by pavreh** [comment]
I installed 2.0.8 from Ubuntu cosmic repository because I wanted to try the latest version. So no self-compilation.

I have installed python-wxgtk3.0 version 3.0.2.0+dfsg-7.

I installed Poedit 2.0.6-1build1 from bionic repository and repeated the bug procedure. The result is the same. Tested with two different po files.

**Comment by vslavik** [comment]
Thanks for confirmation!

**Comment by vslavik** [comment]
Note to self: I got a report of mangled copying when copying *from* Poedit into KDE applications (UTF-8-interpreted-as-Latin1 would show up). Seems likely to be related.
